src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Color {
+    RED, GREEN, BLUE, YELLOW, ORANGE, PURPLE

```

Comment:
To make your enum and code more readable you can list enum items each from the new line

like:
RED, 
GREEN, 
BLUE